radareorg/cutter
# Fixed not switching Disassembly to not opened Graph view using <space> + a bit refactor

### What had to be fixed: ###
Fixes #1268 

### What has been changed: ###
Moved logic behind the handling of the `raisePrioritizedMemoryWidget` signal to the common base class `MemoryDockWidget`. It triggers an action corresponding to the memory widget (Disassembly, Graph, Hexdump, Pseudocode), shows, raises and set focus to it.

### Test plan: ###

- Close Graph dock window;
- Go to disassembly;
- Press space a few times;
- The Graph view should be opened and switching should happen.
